data/common.yaml

```diff
@@ -4,3 +4,4 @@ varnish::conf_file_path: /etc/varnish/varnish.params
 varnish::service::vcl_reload_script: /usr/share/varnish/reload-vcl
 varnish::vcl::vcl_verion: '4'
 varnish::default_version: 6
+varnish::varnish_listen_address: ''
```

Comment:
Remove this since the parameter is optional and has no default value.

Reply:
thanks, removed that